crates/sui-graphql-e2e-tests/tests/stable/prune/prune.exp

```diff
@@ -70,3 +70,57 @@ Response: {
     }
   }
 }
+
+task 12, lines 57-67:
+//# run-graphql
+Response: {
+  "data": {
+    "availableRange": {
```

Comment:
@amnn available range is consistency-available-range, does graphql need another range for non-pruned-range?

Reply:
Yes, we wanted to change the `availableRange` structure to return a kind of nested structure, based on watermarks. It's possible that this is what @wlmyng's PR is about?